docs/developing.md

```diff
@@ -15,3 +15,73 @@ Be sure to write a test for your action in `test/actions/`.
+
+That means that everything necessary to make a release can be done with a few clicks on the GitHub website. All you need to do is make a tagged release at [reuters-graphics/bluprint/releases](https://github.com/reuters-graphics/bluprint/releases), then wait for the computers to handle the job.
+
+Here's how it's done, step by step. The screenshots are from a different repository, but the process is the same.
+
+### 1. Go to the releases page
+
+The first step is to visit [our repository's homepage](https://github.com/reuters-graphics/bluprint) and click on [the "releases" headline](https://github.com/reuters-graphics/bluprint/releases) in the right rail.
+
+![Release button](https://raw.githubusercontent.com/reuters-graphics/bluprint/master/.github/images/releasing-releases-button.png)
+
+### 2. Click 'Draft a new release'
+
+Note the number of the latest release. Click the "Draft a new release" button in the upper-right corner. If you don't see this button, you do not have permission to make a release. Only the maintainers of the repository are able to release new code.
+
+![Draft button](https://raw.githubusercontent.com/reuters-graphics/bluprint/master/.github/images/releasing-draft-button.png)
+
+### 3. Create a new tag
+
+Think about how big your changes are and decide if you're makingg a major, minor or patch release.
+
+All version numbers should feature three numbers separated by the periods, like `1.0.1`. If you're making a major release that isn’t backwards compatible, the latest release’s first number should go up by one. If you’re making a minor release by adding a feature or making a large change, the second number should go up. If you're only fixing bugs or making small changes, the third number should go up.
+
+If you’re unsure, review the standards defined at [semver.org](https://semver.org) to help make a decision. In the end, don’t worry about it too much. Our version numbers don’t need to be perfect. They do need to be three numbers separated by periods.
+
+Once you’ve settled on the number for your new release, click on the "Choose a tag" pull down.
+
+![Tag button](https://raw.githubusercontent.com/reuters-graphics/bluprint/master/.github/images/releasing-tag-button.png)
+
+Enter your version number into the box. Then click the "Create new tag" option that appears.
+
+![Tag dropdown](https://raw.githubusercontent.com/reuters-graphics/bluprint/master/.github/images/releasing-name-tag.png)
+
+### 4. Name the release
+
+Enter the same number into the "Release title" box.
+
+![Name box](https://raw.githubusercontent.com/reuters-graphics/bluprint/master/.github/images/releasing-name-release.png)
```

Comment:
You can actually skip this bit. #5 auto populates that title from the release number.

Reply:
Awesome. This must be a recent UI improvement from GitHub.